src/View/Cell.php

```diff
@@ -203,7 +203,7 @@ public function render($template = null)
             try {
                 return $this->View->render($template);
```

Comment:
I think we can change this line 
```suggestion
                 return $this->View->render($builder->getTemplate());
```

Reply:
That sounds reasonable to me.